lib/spryte/rspec/macros.rb

```diff
@@ -12,7 +12,8 @@ def request(name)
 
         host ENV.fetch("SPRYTE_RSPEC_HOST", "localhost")
 
-        let(:method) { :get }
+        let(:through) { :get }
+        let(:method) { method(:get) }
```

Comment:
My logic goes thus:
1. The reason for leaving `let(:method)` in this code is so that this release doesn't break existing code which might be expecting that `let` to have been set by the call to `request`
2. If that _is_ the case, then we want the code to work, but for the user to see a deprecation warning. This is what the `method` method does.
3. If a user was only ever using `request` (i.e. never calling `method()` directly) then they wouldn't otherwise see any deprecation warnings.

Have I misunderstood what this code is doing?